canonical-web-and-design/maas-ui
# feat(ui): split KVMDetails into Virsh and LXDSingle variants

## Done

- Split `KVMDetails` into `LXDSingleDetails` and `VirshDetails`
- Added placeholder component `LXDClusterDetails` so that the urls in `KVM` all point to the right place

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Check that the LXD and Virsh details pages still work
- Go to `/MAAS/r/kvm/lxd/cluster/1` and check that the placeholder component loads

## Fixes

Fixes canonical-web-and-design/app-squad#354
